rocket-app/library/design/src/main/kotlin/cz/quanti/rocketapp/design/system/RocketAppDimensions.kt

```diff
@@ -9,15 +9,21 @@ data class RocketAppDimensions(
     val sidePadding: Dp,
     val smallSpacer: Dp,
     val mediumSpacer: Dp,
+    val bigSpacer: Dp,
```

Comment:
priklanel bych se k pojmenovani largeSpacer, aby to bylo podle tricek a dalo se pak pouzit treba jeste extraLarge